sbp.bash

```diff
@@ -15,6 +15,23 @@ else
   SBP_TMP=$(mktemp -d) && trap 'command rm -rf "$SBP_TMP"' EXIT;
 fi
 
+# start gitstatus in the background if enabled
+# shellcheck source=src/configure.bash
+source "${SBP_PATH}/src/configure.bash"
```

Comment:
This has a lot of implications. We're now putting all SBP variables into the environment. I'm thinking that either this setup needs to happen within the segment scrift. So if the segment detects that the gitstatus.plugin. isn't running it should start it. Or there could be a hook that does the same thing. In the latter case we will have situations where the segment tries to load, but the plugin isn't running though. So that's probably not ideal. The idea at least is to try to keep the `sbp.bash` loader as clean and unaffected by the segments as possible. WDYT?

Reply:
I just want to check if we need to load gitstatus [here](https://github.com/brujoand/sbp/pull/104/files#diff-2b2e6d680b77581e2aabe6eefb1021f6R23). I don't need any of the other variables but this needs to happen here because the daemon needs to run in the background and we need to keep the variables it exports.

Reply:
I removed this part and moved it into the readme. I tested it locally and the variables are no longer leaked.